knexfile.js

```diff
@@ -0,0 +1,49 @@
+const pg = require("pg");
+pg.defaults.ssl = true;
+
+module.exports = {
+  development: {
+    client: "sqlite3",
+    connection: {
+      filename: "./data/saltyhackers-dev.db3"
+    },
+    useNullAsDefault: true,
+    migrations: {
+      directory: "./data/migrations"
+    },
+    seeds: {
+      directory: "./data/seeds"
+    }
+  },
+
+  production: {
+    client: "pg",
+    connection: process.env.DATABASE_URL || {
```

Comment:
Nice work. Make sure you add a section to the main repo, and store each environment variable you set up. If you use a service that needs an api key to reproduce the project from the source code, add a link to where that can be obtained.